### DISCUSSION TITLE
Collatz Explorers Club

### DISCUSSION THREAD

**Discussion by halso** [Skenvy/Collatz#846] [General]
Hi,

I found your Collatz project at:

https://github.com/Skenvy/Collatz

and am impressed by the Java, Python and Javascript implementations of the Collatz algorithm in the project.  I ran the JUnit tests for the Java implementation and they all passed, and there seems to be comprehensive set of tests.  I also installed the python code using pip and tried the sample functions which worked well.  Unfortunately my npm installation is broken on my mac, so I wasn't able to run the Javascript unit tests, but I've been developing Collatz code using Javascript recently and so aim to get them running at some point.

I’ve recently formed a group called the Collatz Explorers Club.  The group is for amateur mathematicians and/or coders who would like to work with other people on the Collatz conjecture.  The website with more details is at:

[Collatz Explorers Club](https://collatzexplorersclub.wordpress.com/) 

I've been collaborating with someone who is writing Python graphing code for Collatz at https://github.com/musicalmathmind/collatz/tree/main recently.

If Nathan or any other contributors to this project would be interested in joining, please let me know by replying to this on the Discussion or contacting me using the contact form at https://collatzexplorersclub.wordpress.com/contact/

Thanks

Steve.

**Comment by Skenvy** [comment]
Howdy 👋

Sent through a contact form.

Looking forward to collaborating.
